sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateNamespaceExec.scala

```diff
@@ -35,11 +36,14 @@ case class CreateNamespaceExec(
     extends V2CommandExec {
   override protected def run(): Seq[InternalRow] = {
     import org.apache.spark.sql.connector.catalog.CatalogV2Implicits._
+    import org.apache.spark.sql.connector.catalog.SupportsNamespaces._
 
     val ns = namespace.toArray
     if (!catalog.namespaceExists(ns)) {
       try {
-        catalog.createNamespace(ns, properties.asJava)
+        val ownership =
+          Map(PROP_OWNER_NAME -> Utils.getCurrentUserName(), PROP_OWNER_TYPE -> "USER")
```

Comment:
can we do it later? It's unclear to me that what we should do when CREATE TABLE without owner specified.

Reply:
SessionCatalog create table use sparkUser by default

Reply:
ah i see, then let's do it too.

Reply:
that means a namespace always have the owner properties?

Reply:
yes, as earlier discussion, we let spark side to define the ownership and leave whether to expose ownership api latter